lib/gollum/app.rb

```diff
@@ -234,7 +234,7 @@ class App < Sinatra::Base
         wikip = wiki_page(params[:splat].first)
         halt 500 if wikip.nil?
         wiki   = wikip.wiki
-        page   = wiki.paged(join_page_name(wikip.name, wikip.ext), wikip.path, exact = true)
+        page   = wikip.page
```

Comment:
This line until now was incredibly inefficient: `wikip` already contains the `page`, but we were doing *another* call to `wiki.paged` nevertheless! There were a couple of instances of this. Should improve page load times a bit.